spec/01-lexical-syntax.md

```diff
@@ -25,6 +25,9 @@ classes (Unicode general category given in parentheses):
 1. Operator characters. These consist of all printable ASCII characters
    (`\u0020` - `\u007E`) that are in none of the sets above, mathematical
    symbols (`Sm`) and other symbols (`So`).
+1. [Bidirectional explicit formatting](https://www.unicode.org/reports/tr9/#Bidirectional_Character_Types)
+   characters. The nine characters `\u202a - \u202e` and `\u2066 - \u2069`, 
+   inclusive. These are forbidden from appearing in character and string literals.
```

Comment:
That's not correct, the characters are not allowed at all in source files. In string / character literals, the characters can be included as unicode escapes, but not literally.

In 2.12 (just for the record), the characters are also not allowed at all. The difference is that in unicode escapes (of these forbidden characters) are allowed everywhere (not only in character / string literals) in 2.12.

Reply:
I've changed this. It now seems a bit odd that this is in a list prefaced by saying "To construct tokens...". Do you think it should be moved to the first line of the section ("Scala source code consists of Unicode text" [and these characters are allowed])?

Reply:
Then do you think the other file (13-syntax-summary.md) doesn't need to be changed at all, and adding this note in this file is enough?

Reply:
For me that would be enough, but maybe others would prefer also a not in the syntax summary file? Things are not very precise already, for example `stringElement ::= charNoDoubleQuoteOrNewline | escapeSeq` but `charNoDoubleQuoteOrNewline` is not actually defined in the spec as far as I can tell...

Reply:
Yes, I noted the implicitly-defined xxxOrYyy constructions in the syntax summary (there are several of them) in a previous comment.

FWIW I would prefer a note in the syntax summary; that's where I looked first (and I was surprised that the summary is not an exact subset of the full grammar's productions).

Reply:
I pushed a new commit that makes only that change to both files and does not change any grammar productions. What do you think?

Reply:
To me that looks good, thank you!